GooglePlayInstant/Editor/PlayInstantQuickDeployWindow.cs

```diff
@@ -68,14 +69,29 @@ void OnGUI()
         private void OnGuiCreateBundleSelect() 
         {
             EditorGUILayout.LabelField("AssetBundle Creation", EditorStyles.boldLabel);
-            EditorGUILayout.LabelField("Use the Unity Asset Bundle Browser to select your game's main scene " + 
-                "and bundle it (and its dependencies) into an AssetBundle file.", EditorStyles.wordWrappedLabel);
+            EditorGUILayout.LabelField("Use the Unity AssetBundles Browser to select your game's main scene " + 
+                                       "and bundle it (and its dependencies) into an AssetBundle file.", EditorStyles.wordWrappedLabel);
             EditorGUILayout.Space();
-            EditorGUILayout.LabelField(string.Format("AssetBundle Browser version: {0}", "not found"), EditorStyles.wordWrappedLabel);
+            EditorGUILayout.LabelField($"AssetBundle Browser version: { AssetBundleBrowserClient.BundleBrowserVersion }",  EditorStyles.wordWrappedLabel);
```

Comment:
Does https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated work with .NET 3.5? Make sure you test with that version instead of 4.X

Name "Asset Bundle Browser"

Reply:
Resolved. Now working in 3.5 to make sure that everything is compatible